Examples/UIkit/TestCountryPickerFramework/PickerViewController.swift

```diff
@@ -58,6 +58,8 @@ class PickerViewController: UIViewController {
     }
     
     private func setupStoryboardPickerViewCallback() {
+        // Use setSelectedCountry to set a default country code
+        storyboardPickerView.setSelectedCountry(Country(countryCode: "GB"))
```

Comment:
why hardcode ?

Reply:
@SURYAKANTSHARMA This is the sample code. Code user can pass in the required `countryCode`.

Reply:
got it